R/merPredict.R

```diff
@@ -254,6 +267,41 @@ predictInterval <- function(merMod, newdata, which=c("full", "fixed", "random",
   ##Calculate yhat as sum of the components (fixed plus all groupling factors)
   fe.tmp <- fixef(merMod)
   vcov.tmp <- as.matrix(vcov(merMod))
+  if (fix.intercept.variance) {
+    #Assuming all random effects include intercepts.
+    intercept.variance = vcov.tmp[1,1]
+
+    groupsizes = ngrps(merMod)
+    for(j in names(groupsizes)){ #for every group of random e
```

Comment:
Is it problematic that we are iterating over `j` in nested loops (lines 152 and 275) or is this on purpose?